x/examples/outline-connectivity/main.go

```diff
@@ -150,6 +153,37 @@ func main() {
 			if err != nil {
 				log.Fatalf("Failed to output JSON: %v", err)
 			}
+			// Send error report to collector if specified
+			if !success && *collectorFlag != "" {
+				jsonData, err := json.Marshal(record)
```

Comment:
Can you provide some `SendReport(record, collectorAddress)` function, for readability?

Reply:
The commit 777a241f57d1550f34017075ea4709bd3d3d078d improved readability